tamr_toolbox/utils/client.py

```diff
@@ -85,6 +90,61 @@ def create(
     return client
 
 
```

Comment:
Can you add the version check here, like with movement.py and jobs.py? i.e. @requires_tamr_version(min_version="2020.004.0")
def _run_custom(

Reply:
Changed accordingly